portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,44 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function LoadData() {
+    fetch('/data').then(response => response.json()).then((stats) => {
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    const statsListElement = document.getElementById('comment-container');
+
+    statsListElement.innerHTML = '';
+    stats.forEach((comment) => {
+        statsListElement.appendChild(
+            createCommentElement(comment));
+    })
+  });
+}
+
+function createCommentElement(comment) {
```

Comment:
please keep indent style consistent, maybe you can refer to style guide for suggestion

Reply:
fixed in 41f0ac